### PR TITLE
[neutron] Switch to Neutron Lbaas v2 for Newton release

### PR DESCRIPTION
**Pull request by mjura**
In OpenStack Newton release Neutron Lbaas v1 is deprecated.
We have to remove Lbaas v1 from our configuration.

### CODE REVIEW

**Comment by toabctl** [comment]
nitpick: Can you use ```neutron:``` and ```ceilometer:``` instead of the names in brackets please?

**Comment by matelakat** [comment]
> why are you deleting migration 101?

@dirkmueller There were 2 migrations with number 101 for some reason - I guess that might be the reason?.

**Comment by mjura** [comment]
> why are you deleting migration 101?

There were 2 migration scripts with 101 and this which I'm removing is not needed anyway

**Comment by matelakat** [comment]
@dirkmueller could you please give it another review? Migration 102 was renamed, because we had 2 of them for some reason.

**Comment by toabctl** [comment]
@mjura this needs a rebase

**Comment by mjura** [comment]
@toabctl : PR was rebased :)

**Comment by toabctl** [comment]
@dirkmueller any more comments?

**Comment by nicolasbock** [comment]
@dirkmueller could you have another look please?

**Comment by nicolasbock** [comment]
@dirkmueller ?

**Comment by mjura** [comment]
It is ready to merge

**Comment by mjura** [comment]
@dirkmueller: do you have still some comments?

File: crowbar_framework/app/helpers/barclamp/neutron_helper.rb [hunk 1]
```diff
@@ -63,6 +63,16 @@ def networking_ml2_type_drivers_default_tenant_network_for_neutron(selected)
                          selected.to_s)
     end
 
+    def networking_lbaasv2_drivers(selected)
+      options_for_select(
+        [
+          ["haproxy", "haproxy"],
```
Comment (dirkmueller) [inline crowbar_framework/app/helpers/barclamp/neutron_helper.rb]: official project name spelling is HAProxy

File: crowbar_framework/app/helpers/barclamp/neutron_helper.rb [hunk 2]
```diff
@@ -63,6 +63,16 @@ def networking_ml2_type_drivers_default_tenant_network_for_neutron(selected)
...
+          ["f5", "f5"]
```
Comment (dirkmueller) [inline crowbar_framework/app/helpers/barclamp/neutron_helper.rb]: Same here, its F5

File: crowbar_framework/app/views/barclamp/neutron/_edit_attributes.html.haml [hunk 3]
```diff
@@ -33,6 +33,23 @@
...
         = boolean_field :use_dvr
 
+      #lbaas_container
+        = boolean_field :use_lbaas
+
+      #lbaasv2_driver_container
+        = select_field :lbaasv2_driver, :collection => :networking_lbaasv2_drivers
+
+      #f5_driver_container
+        = string_field %w(f5 icontrol_hostname)
```
Comment (dirkmueller) [inline crowbar_framework/app/views/barclamp/neutron/_edit_attributes.html.haml]: those are not in the template.. rebase error ?

Comment (mjura) [inline crowbar_framework/app/views/barclamp/neutron/_edit_attributes.html.haml]: I fixed this

**Comment by mjura** [comment]
> The schema migration renumbering looks odd, why are you removing 101 migration?

Problem with 101 migration script is that it is already overlapping with script `101_add_max_pool_size.rb`, so there is mistake. Because we are removing lbaas v1 and we will have only one lbaas agent so there is no need to have `use_lbaas` and `use_lbaasv2` both attributes. 

If you have some other idea on this, let me know, I am open for other solutions.

**Comment by mjura** [comment]
I rebased this change

**Comment by mjura** [comment]
It is ready to merge

**Pull request merged**